### ISSUE TITLE
ASReview Analysis submodule struggles with simulations with exactly 1 inclusion. 

### ISSUE THREAD

**Issue by J535D165** [asreview/asreview#499]
More specific: If the number of relevenant items in the dataset matches the number of relevant items in the prior knowledge, the submodule ASReview Analysis struggles with computing metrics. This pops up when using ASReview-statistics. 

Example: Dataset with 1 relevant record which is used ats prior knowledge. Simulate run and then call `asreview stats`. 

```
jonathan$ asreview stat test.h5 
/Users/jonathan/asreview/asreview/analysis/analysis.py:170: RuntimeWarning: invalid value encountered in true_divide
  norm_yr = (np.array(self.inc_found[fl]["avg"]) - dy) / y_norm
/Users/jonathan/asreview/asreview/analysis/analysis.py:171: RuntimeWarning: invalid value encountered in true_divide
  norm_y_err = np.array(self.inc_found[fl]["err"]) / y_norm
Traceback (most recent call last):
  File "/Users/jonathan/.pyenv/versions/asreview/bin/asreview", line 11, in <module>
    load_entry_point('asreview', 'console_scripts', 'asreview')()
  File "/Users/jonathan/asreview/asreview/__main__.py", line 80, in main
    entry.load()().execute(sys.argv[2:])
  File "/Users/jonathan/asreview-statistics/asreviewcontrib/statistics/entrypoint.py", line 71, in execute
    print(stat)
  File "/Users/jonathan/asreview-statistics/asreviewcontrib/statistics/statistics.py", line 99, in __str__
    results = self.to_dict()
  File "/Users/jonathan/asreview-statistics/asreviewcontrib/statistics/statistics.py", line 89, in to_dict
    "loss": self.loss(),
  File "/Users/jonathan/asreview-statistics/asreviewcontrib/statistics/statistics.py", line 61, in loss
    return loss/len(avg_discovery_time)
ZeroDivisionError: division by zero

```

**Comment by Rensvandeschoot** [comment]
Indeed, it should result in a warning

**Comment by J535D165** [comment]
Resolved in #688
